Podfile.lock

```diff
@@ -1,20 +1,24 @@
 PODS:
-  - MessageKit (0.13.3)
+  - MessageKit (1.0.0)
   - openssl-ios-bitcode (1.0.210)
+  - SnapKit (4.0.1)
 
 DEPENDENCIES:
-  - MessageKit
-  - openssl-ios-bitcode
+  - MessageKit (= 1.0.0)
+  - openssl-ios-bitcode (= 1.0.210)
+  - SnapKit (~> 4.0.0)
```

Comment:
what's the license of SnapKit?

Reply:
https://github.com/SnapKit/SnapKit/blob/develop/LICENSE

Reply:
k, that's MIT, fine.